documentation/Troubleshooting.md

```diff
@@ -927,6 +930,38 @@ spec:
         - '--disable-full-test'
 ```
 
+## CoreDNS Cannot Resolve the Name
+
+### Case 1
+
+**Symptoms**: A pod can't resolve the non-FQDN. Check inside the pod looks the following:
+
+```
+$ nslookup kubernetes.default
+Server:         172.30.0.10
+Address:        172.30.0.10:53
+
+
+** server can't find kubernetes.default: NXDOMAIN
+```
+
+**Root cause**: Images that are based on Alpine Linux do not use `search` directives in `/etc/resolv.conf` by default.
```

Comment:
can we put an Alpine version which has such behaviour and find some explanation why: design intent or a bug/gap ?

Reply:
Sure, I'll extend description